atom3/pair.py

```diff
@@ -97,7 +97,7 @@ def complex_to_pairs(complex, get_pairs, output_dir):
                  .format(name, len(pairs), num_subunits))
     sub_dir = output_dir + '/' + db.get_pdb_code(name)[1:3]
     f = name
-    if 'mut' in f:
+    if ('mut' in f) and ('mut' not in db.get_pdb_code(name)):
         pdb = db.get_pdb_code(name)(f) + f[f.rfind('_') + 1: f.find('.')]
```

Comment:
This line `db.get_pdb_code(name)(f)` is also suspicious, since `f` is used as argument to `db.get_pdb_code(name)` which should have returned a str. I don't know what is the intended behavior, so I just leave it as is.

Reply:
@YianniL do you know what this line should be?